json/src/test/java/com/basistech/rosette/dm/json/plain/JsonTest.java

```diff
@@ -324,6 +327,17 @@ public void oneWithEverythingOldEntities() {
         keyphraseBuilder.add(keyphrase);
         builder.keyphrases(keyphraseBuilder.build());
 
+        TransliterationResults.Builder transliterationBuilder =
+                new TransliterationResults.Builder(LanguageCode.ENGLISH,
+                        ISO15924.Latn,
+                        THIS_IS_THE_TERRIER_SHOT_TO_BOSTON);
+
+        // Translation: This is the terrier shot to boston
+        transliterationBuilder.addTransliteration(
+                LanguageCode.ARABIC, Transliteration.of(ISO15924.Arab, "هذا هو النار ضربة الى بوسطن."));
```

Comment:
This string is not a transliteration so it is weird to put it in the transliteration slot, though it doesn’t invalidate the test.

This should use the constant `THIS_IS_THE_TERRIER_SHOT_TO_BOSTON_ARABIC` instead of duplicating the string.